examples/warm_up_example/warmed_up_fedprox/client.py

```diff
@@ -29,19 +28,17 @@ def __init__(
         data_path: Path,
         metrics: Sequence[Metric],
         device: torch.device,
-        pretrained_model_dir: Path,
+        pretrained_model_path: Path,
```

Comment:
Moving to just specifying the model path rather than trying to infer it based on the client name, which can not longer be fixed by the random seed (they use UUIDs not under the hood).